### PR TITLE
Handle Chrome OS 105 WebUSB error

### PR DESCRIPTION
**Pull request by microbit-robert**

### CODE REVIEW

**Comment by microbit-robert** [comment]
I was unsure about changing device.ts where ConnectionStatus is typed and webusb.ts where ConnectionStatus is initially set for something potentially short term, but happy to discuss.

File: src/project/project-actions.tsx [hunk 1]
```diff
@@ -1049,6 +1062,11 @@ export class ProjectActions {
       { changeName: change.name }
     );
   };
+
+  private isChromeOS105(): boolean {
+    const userAgent = navigator.userAgent;
+    return /(CrOS)/.test(userAgent) && /(Chrome\/105)/.test(userAgent);
```
Comment (microbit-matt-hillsdon) [inline src/project/project-actions.tsx:1068]:
```suggestion
    return /CrOS/.test(userAgent) && /Chrome\/105\b/.test(userAgent);
```

I think the round brackets serve no purpose. Add \b (word boundary) so, um, I don't need to worry about Chrome 1050 (!)

**Review by microbit-matt-hillsdon — APPROVED** [review]
LGTM but @microbit-robert please review our changes from this morning.

**Comment by microbit-robert** [comment]
> LGTM but @microbit-robert please review our changes from this morning.

All looks good after latest changes.

**Pull request merged**